flang/lib/Optimizer/OpenMP/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+get_property(dialect_libs GLOBAL PROPERTY MLIR_DIALECT_LIBS)
+
+add_flang_library(FlangOpenMPTransforms
```

Comment:
as we have 
libflangFrontend.a
libflangFrontendTool.a

maybe Flang => flang to be consistent ? thanks